app/swagger.ts

```diff
@@ -24,15 +26,11 @@ export const swaggerDocument = {
   },
   servers: [
     {
-      url: 'https://coronastatus.no/api',
+      url: `https://${BASE_URL}${urls.api}`,
       description: 'Production environment'
     },
     {
-      url: 'https://dev.coronastatus.no/api',
-      description: 'DEV Env'
-    },
-    {
-      url: 'http://localhost:7272/api',
+      url: `http://localhost:7272${urls.api}`,
```

Comment:
We use a different IP, but not very important right now.

Reply:
Yeah but if people clone the repo and use the README it will run on 7272.